state/accountsDB.go

```diff
@@ -1282,8 +1285,8 @@ func (adb *AccountsDB) processSnapshotCompletion(
 		return
 	}
 
-	err := trieStorageManager.Remove([]byte(lastSnapshotStarted))
-	handleLoggingWhenError("could not remove lastSnapshotStarted", err, "rootHash", rootHash)
+	err := trieStorageManager.RemoveFromAllEpochs([]byte(lastSnapshot))
```

Comment:
I think this works, as seen in the following scenario:

Epoch 100 -> snapshot started (takes a long time to finalize it) `lastSnapshot` written in epoch 100
Epoch 101 -> snapshot skipped `lastSnapshot` written in epoch 101
Epoch 102 -> snapshot skipped `lastSnapshot` written in epoch 102
Epoch 100 snapshot finished, removed all `lastSnapshot` entries from all opened epochs, marks the Epoch 100 as active.
*** restarts the node ***
The node searches the ActiveDB marker, opens Epoch 100 & one ore more previous epochs and all the DBs up until Epoch 102. Then, it will start the snapshot from Epoch 102.

Alternatively, if the node would have been restarted before the removal of the `lastSnapshot` marker, it would have opened more DBs (until it could have found 2 ActiveDB markers) & start the snapshot from the most recent Epoch (102) not the uncompleted Epoch 100